web/stats.php

```diff
@@ -53,16 +69,18 @@
 require_once "../phplib/questionnaire_report_${year}_WMC.php";
 
 $rep_info = array();
-$voting_areas = mapit_call('postcode', $postcode, array(), array(
+$area_representatives = array();
+$area_name = '';
+$voting_areas = mapit_call('postcode', $postcode, $mapit_args, array(
```

Comment:
I realise this doesn't change behaviour, but it looks like this call happens even if the postcode is blank, which seems a bit superfluous :)